lib/fork.js

```diff
@@ -18,13 +18,29 @@ module.exports = function (args) {
 
 	var ps = childProcess.fork(babel, args, options);
 
+	var killed = false;
+	var killedPromise = new Promise(function (resolve) {
+		ps.on('exit', function (code) {
+			killed = true;
+			resolve(code);
+		});
+	});
+
+	function kill() {
+		if (!killed) {
+			ps.kill();
+		}
+		return killedPromise;
+	}
+
 	var promise = new Promise(function (resolve, reject) {
 		ps.on('results', function (results) {
+			results.kill = kill;
```

Comment:
I think it would be better to attach `kill()` method to a returned Promise, like `on()` - https://github.com/jamestalmage/ava/blob/jt-kill-forked-processes/lib/fork.js#L66.

That way, we won't have weird stuff in test results.